### PR TITLE
fix(DB/Quest): "Plundering the Plunderers" has incorrect quest completion text

### PR DESCRIPTION
**Pull request by Asthalor**
The quest reward gossip had text with a setted name and gender. Its changed to use the name and character now

Closes AzerothCore issue #6976

## Changes Proposed:
-  Changing in the database the text of the quest using $N to take the name and $gman:woman; to change between man and woman.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6976
- Closes https://github.com/chromiecraft/chromiecraft/issues/1219

## SOURCE:

Picture of the new text

image

With man named World

image

## Tests Performed:

-  .go c id 7161
- .quest add 2381 to add the quest
- .quest complete 2381 to complete it
-  I talked with the goblin and saw the he used my name and gender on the text.

## How to Test the Changes:

1. .go c  id 7161
2. .quest add 2381 to add the quest
3. .quest complete 2381 to complete it
4. Talk with the goblin and see if the text use the name and gender of the character.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1626545461944168600.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1626545461944168600');
+
+-- We change the quest reward to use the name and gender of our character.
+UPDATE `quest_offer_reward` SET `RewardText` = 'Incredible! Improbable! Simply amazing! You\'ve got talent, $N. Either that or you\'re the luckiest $gman:woman; alive!$b$bHere\'s your cut of the action. I\'m sure you would make better use of this stuff than I could.$b$b' WHERE (`ID` = 2381);
+
+
```
Comment (LukasVolgger) [inline data/sql/updates/pending_db_world/rev_1626545461944168600.sql:6]: One line too many

**Review by LukasVolgger — APPROVED** [review]

**Comment by LukasVolgger** [comment]
Btw the title refers to a gossip, but you have fixed a quest

**Comment by Asthalor** [comment]
Woops, yeah you are right. Title changed.

**Comment by LukasVolgger** [comment]
I ran the script and everything is fine. No further tests needed because you already tested it ingame.

**Comment by syssneck** [comment]
Character name and gender are now shown correctly in the quest completion text:
unknown_2021 07 19-09 35

**Pull request merged**